provider/data_source_keycloak_realm.go

```diff
@@ -60,6 +60,10 @@ func dataSourceKeycloakRealm() *schema.Resource {
 				Type:     schema.TypeBool,
 				Computed: true,
 			},
+			"ssl_required": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
 
```

Comment:
Changes in this file are not really related to the PR, but I saw they were missing, so I added them. see also #218